modules/phase_field/tests/phase_field_kernels/tests

```diff
@@ -60,4 +60,11 @@
     input = 'ConstructionOrder.i'
     exodiff = 'ConstructionOrder_out.e'
   [../]
+
+  [./SimpleSplitCHWRes]
+    type = 'Exodiff'
+    input = 'SimpleSplitCHWRes.i'
+    exodiff = 'SimpleSplitCHWRes_out.e'
+    max_parallel = 1                
```

Comment:
This is fine for a test, sometimes "lu" is the fastest way to get something done, but it doesn't work in parallel so you can leave it for now.